setup.py

```diff
@@ -2,8 +2,34 @@
 
 import os
 import re
+import sys
 import codecs
+import sys
+import importlib
 from setuptools import setup, find_packages
+from setuptools.command.build_ext import build_ext
```

Comment:
I think all of this is depreciated these daze because
> `python setup.py` and the use of `setup.py` as a command line tool are deprecated.
* https://packaging.python.org/en/latest/discussions/setup-py-deprecated/